airflow/www/decorators.py

```diff
@@ -62,38 +61,28 @@ def wrapper(*args, **kwargs):
     return cast(T, wrapper)
 
 
-def gzipped(f: T) -> T:
+def compressed(f: T) -> T:
```

Comment:
Can't we just use the decorator from flask_comrpressed rather than having to implement our own? https://github.com/colour-science/flask-compress/blob/master/flask_compress.py#L206-L215

Reply:
This decorator requires that we have a global variable with a Compress instance. I preferred to avoid it, so I implemented this decorator again. See: https://github.com/colour-science/flask-compress/issues/12